c7n/resources/kms.py

```diff
@@ -41,15 +43,31 @@ def augment(self, resources):
 
 class DescribeKey(DescribeSource):
 
+    def get_resources(self, ids, cache=True):
+        # this forms a threshold beyond which we'll fetch individual keys of interest.
```

Comment:
Was this meant to be commented out for future implementation? It seems that the baseclass is doing a somewhat similar filter in `query.get`. For the threshold part, what is the significance of `len(ids) < 10`?

Reply:
yeah.. there's some work in a separate pr on handling this across a few resources missing it. the nutshell is avoiding any client filtering on fetch, but we're trading off against doing individual api calls to resources to fetch by id.